File: 백준/Bronze/2581. 소수/소수.cc
#include <stdio.h>
#include <stdbool.h>

bool is_prime(int n) {
    if (n < 2) return false;
    for (int i = 2; i * i <= n; i++) {
        if (n % i == 0) return false;
    }
    return true;
}

int main() {
    int m, n;
    scanf("%d%d", &m, &n);

    int sum = 0, min_prime = -1;
    bool found = false;
    for (int i = m; i <= n; i++) {
        if (is_prime(i)) {
            sum += i;
            if (!found || i < min_prime) {
                min_prime = i;
                found = true;
            }
        }
    }

    if (found) {
        printf("%d\n%d\n", sum, min_prime);
    } else {
        printf("-1\n");
    }

    return 0;
}

File: 백준/Bronze/2581. 소수/소수.py
M = int(input())
N = int(input())


def is_prime(n):
    if n == 1:
        return False

    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False
    return True


result = [i for i in range(M, N + 1) if is_prime(i)]

if result:
    print(sum(result))
    print(min(result))
else:
    print(-1)